pkg/activator/handler/handler.go

```diff
@@ -109,9 +106,11 @@ func (a *ActivationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			})
 			success = (err == nil) && httpStatus == http.StatusOK
 		}
+		attempts := 0
 		if success {
 			// Once we see a successful probe, send traffic.
-			attempts, httpStatus = a.proxyRequest(w, r, target)
+			httpStatus = a.proxyRequest(w, r, target)
+			attempts = 1
```

Comment:
Should we report the retries of the networking probe instead to be able to monitor that?

Related to that: Should the `httpStatus == http.StatusServiceUnavailable` case rather log a warning vs. an Error?

Reply:
I made both error paths `logger.Warnx` calls and changed the attempts to count probes + the request.